src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -6309,10 +6309,14 @@ namespace InternalHeatGains {
                     TAirIn = TRecirc * RecircFrac + TSupply * (1.0 - RecircFrac);
                     WAirIn = WRecirc * RecircFrac + WSupply * (1.0 - RecircFrac);
                 } else if (AirConnection == ITEInletRoomAirModel) {
-                    // Room air model option not implemented yet
+                    // Room air model option: TAirIn=TAirZone, according to EngineeringRef 17.1.4 (Yanfei Li, 09/05/2019)
```

Comment:
And we don't tag lines with name and date - we let git history track that. Here and below.

Reply:
oh thanks. this is not urgent. We can wait after 9.2 release. I will fix the format issue then.